> 8-1 확인 문제
- 1번 : 1 (cons a = 10
            console.log(a * a))
- 2번 : 3
- 3번 : 3(let numer = NEW Number(10))

> 8-2 확인 문제
- 1번 : 4(throw)
- 2번 : 1
- 3번 : try 구문입니다. catch 구문입니다. finally 구문입니다.
